qiskit/pulse/macros.py

```diff
@@ -30,6 +45,13 @@ def measure(
     """Return a schedule which measures the requested qubits according to the given
     instruction mapping and measure map, or by using the defaults provided by the backend.
 
+    If the backend has an attribute ``target``, the function uses the measurement logic,
+    "_measure_v2" that takes ``target`` of the ``backend``, ``meas_map`` and ``qubit_mem_slots``
+    assignment.
+    Otherwise, if the backend is None or ``backendV1``, the function uses the
+    measurement logic, "_measure_v1" including ``instruction_schedule_map`` and ``meas_map``
+    as inputs.
```

Comment:
```suggestion
    .. note::
        This function internally dispatches schedule generation logic depending on input backend model.
        For the :class:`.BackendV1`, it considers conventional :class:`.InstructionScheduleMap` 
        and utilizes the backend calibration defined for a group of qubits defined in the `meas_map`.
        For the :class:`.BackendV2`, it assembles calibrations of single qubit measurement 
        defined in the backend target to build a composite measurement schedule for `qubits`.
```
This doesn't need to be written here. Note that this is API document that end-users read, so implementation details must be avoided.

Reply:
I fixed at https://github.com/Qiskit/qiskit-terra/pull/9987/commits/402984987df12b4fc46459db1a57bf8a80d35486.